### ISSUE TITLE
Add support for composer

### ISSUE THREAD

**Issue by ayham-hassan** [amar71g/Janna#1]
This library is fine, but it requires me to download it and move my exists project that have another dependencies to be satisfied with it!. 

I think it is better to be supported to be library framework that can be added via dependency managers such as composer. 

You can give the developer both choices. add the framework as a library or  just create an installer that configure all of things such as Laravel Framework.

**Comment by amar71g** [comment]
Of course, but in the issuance of a stable version of the stage will be included for the library Thanks
